load-test-framework/src/main/java/com/google/pubsub/flic/controllers/mode/Mode.java

```diff
@@ -0,0 +1,70 @@
+package com.google.pubsub.flic.controllers.mode;
+
+import com.google.auto.value.AutoValue;
+
+import com.google.protobuf.Duration;
+import com.google.protobuf.util.Durations;
+
+import java.util.Optional;
+
+@AutoValue
+public abstract class Mode {
```

Comment:
Maybe "TestParameters" instead of "Mode"?

Reply:
Done.